pure.zsh

```diff
@@ -331,6 +331,20 @@ prompt_pure_async_git_arrows() {
 	command git rev-list --left-right --count HEAD...@'{u}'
 }
 
+# Try to lower the priority of the worker so that disk heavy operations
+# like `git status` has less impact on the system responsivity.
+prompt_pure_async_renice() {
+	setopt localoptions noshwordsplit
+
+	if command -v renice >/dev/null; then
+		command renice +15 -p $$
```

Comment:
```suggestion
		command renice +15 --pid $$
```

for readability

Reply:
These long flags won’t be available e.g. on `busybox` systems, perhaps others. would result in error. The short ones at least are pretty universal.

Reply:
Should I revert to short names, then?

Reply:
I think that would be best. Otherwise I think we will start seeing bug reports about the errors it would cause on such systems.

Reply:
done